roles/power_aix_bootstrap/README.md

```diff
@@ -11,9 +11,32 @@ None.
 
 Available variables are listed below, along with default values:
 
-    pkgtype (True, str, yum)
+    pkgtype (True, str, none)
```

Comment:
Find if the True argument can be replace for another more meaningful one and also if none is the right value.

Reply:
There is not a default way to show which values are required, type or values. 
I added more elaborated description of the arguments, values and descriptions.

Reply:
@pvtorres found anything on this?